server/src/errorUtils.ts

```diff
@@ -41,6 +41,10 @@ export function handleError(
     trace.exit({ vscodeMessage });
 }
 
+export function isCancellationError(value: unknown): boolean {
```

Comment:
Use this signature instead:
`export function isCancellationError(value: unknown): value is PQP.CommonError.CancellationError`

Also, would it be useful to add a isCancellationError to PQP?